scripts/local_runner.py

```diff
@@ -11,13 +11,21 @@ def run_cmd(opts: argparse.Namespace, run_id: int) -> str:
     log_file = os.path.join(log_dir, f"run{run_id}.txt")
     main_ns = "erp12.cbgp-lite.benchmark." + opts.search
     suite_ns = "erp12.cbgp-lite.benchmark.suite.psb"
+    types_file = os.path.join(log_dir, f"run{run_id}_types.edn")
     return "; ".join(
         [
             f'echo "Starting run {run_id}"',
             "export PATH=$PATH:/usr/java/latest/bin",
             f"cd {opts.cbgp}",
             f"mkdir -p {log_dir}",
-            f"{opts.clj} -X:benchmarks {main_ns}/run :suite-ns {suite_ns} :data-dir '\"{opts.data_dir}\"' :problem '\"{opts.problem}\"' 2>&1 | tee {log_file}",
+            (
+                f"{opts.clj} -X:benchmarks {main_ns}/run "
+                f":suite-ns {suite_ns} "
+                f":data-dir '\"{opts.data_dir}\"' "
+                f":problem '\"{opts.problem}\"' "
+                f":type-counts-file '\"{types_file}\"'" if opts.log_types else ""
+            ),
+            f"2>&1 | tee {log_file}",
```

Comment:
This introduced a bug that made it so logs are empty, because this is after the `;` that ends the clj command.

Reply:
Good catch! I'll move that line into the parentheses.